media-libs/gstreamer/gstreamer-1.22.1.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit gstreamer-meson pax-utils
+
+DESCRIPTION="Open source multimedia framework"
+HOMEPAGE="https://gstreamer.freedesktop.org/"
+SRC_URI="https://${PN}.freedesktop.org/src/${PN}/${P}.tar.xz"
+
+LICENSE="LGPL-2+"
+SLOT="1.0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~x64-solaris ~x86-solaris"
+IUSE="+caps +introspection +orc unwind"
+
+RDEPEND="
+	>=dev-libs/glib-2.56.0:2[${MULTILIB_USEDEP}]
```

Comment:
Have you checked through dependency changes in all the tarballs?
For example glib dep was raised from 2.56.0 to 2.62.0 in all the packages I looked - which in turn highlights we haven't been updating the eclass, which depends on some gst-1.14 era glib, but as long as media-libs/gstreamer pulls in a new enough one, that doesn't matter in practice.
Of course no-one is going to have older than 2.62.0 installed anyways, but seems like the deps weren't checked besides the bump script run maybe?